properties/composite.json

```diff
@@ -4,7 +4,73 @@
             "secondary": {"value": "{color.base.30.value}"}
-        }
+        },
+    
+        "style":{   
+            "h1":{
+                "font-family":{"value":"{font.family.serif.value}"},
+                "size": {"value": "{dimension.font.size.xxxlarge.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.regular.value}"}
+                },
+            
+            "h2":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.xxlarge.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.bold.value}"}
+                },
+    
+            "h3":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.xlarge.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.bold.value}"}
+                },
+    
+            "h4":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.large.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.bold.value}"}
+                },
+            
+            "h5":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.large.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.bold.value}"}
+                },
+
+            "h6":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.medium.value}"},
+                "line-height": {"value": "{font.line-height.small.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.regular.value}"}
+                },
+
+            "body":{
+                "font-family":{"value":"{font.family.sans.value}"},
+                "size": {"value": "{dimension.font.size.medium.value}"},
+                "line-height": {"value": "{font.line-height.medium.value}"},
+                "color": {"value": "{text.color.base.10.value}"},
+                "font-weight":{"value": "{font.weight.regular.value}"}
+                },
+            
+             "description":{
```

Comment:
I'm not sure what elements to apply this style to, can you give me an example of a description in a page?